ground/src/main/java/com/google/android/ground/ui/offlinebasemap/viewer/OfflineAreaViewerFragment.kt

```diff
@@ -63,7 +62,7 @@ class OfflineAreaViewerFragment @Inject constructor() : Hilt_OfflineAreaViewerFr
   override fun getMapViewModel(): BaseMapViewModel = viewModel
 
   private fun panMap(offlineArea: OfflineArea) {
-    mapFragment.viewport = offlineArea.bounds.toModelObject()
+    mapFragment.viewport = offlineArea.bounds
```

Comment:
I might be missing something, but where do we actually perform the conversion back to GMS objects in the Map Fragment? Seems like we currently don't do this anywhere?

Reply:
Only when we are setting the camera position or checking for points within a bound.